src/scout_apm/core/config.py

```diff
@@ -212,7 +212,7 @@ class Defaults(object):
     def __init__(self):
         self.defaults = {
             "app_server": "",
-            "application_root": "",
+            "application_root": os.getcwd(),
```

Comment:
Is this directly related to errors, or should this be a separate change? What are the potential liabilities for defaulting this to CWD?

Reply:
This change could be pulled into a separate PR. Having the noise of the full path in the error stack trace elevated the priority of #416 as the stack trace is a big part of the error monitoring product.

The risks with this is that it will base `application_root` based on how the application is run rather than how the application is structured. If someone runs the app from a parent directory, the directories between the code and where the application is being run will cause additional directories to appear in stack traces within the Scout app.

Reply:
@dlanderson Are you good with the above risk? We could put the working directory as another value, release it and then review the values to see if it would cause any issues.

Reply:
I think it's good - it should be an improvement for anyone not using django where we can use BASE_DIR